contents/handbook/engineering/oncall.md

```diff
@@ -12,14 +12,20 @@ Make sure you've downloaded the PagerDuty app for your phone, and that you've se
 
 ## What to do when you get paged
 
-A chunk of our high-frequency alerts have runbooks attached, most of which live [in the runbooks section](https://posthog.com/docs/runbook) of our docs. The runbook should tell you what to look at and easy fixes if there are any. Every alert also has a link to the grafana graph that triggered the alert.
+A chunk of our high-frequency alerts have runbooks attached, most of which live [in the public runbooks section](https://posthog.com/docs/runbook) of our docs or in [our private runbooks](http://go/runbooks). The runbook should tell you what to look at and easy fixes if there are any. Every alert also has a link to the grafana graph that triggered the alert.
```

Comment:
most is in the private runbooks and we should move everything there or link to the public ones as needed. Historically it was public due to self-hosting, but we can't write everything publicly and a single place is best for speed.